src/openrct2/object/ObjectFactory.cpp

```diff
@@ -104,15 +104,20 @@ class ReadObjectContext : public IReadObjectContext
     }
 
     ReadObjectContext(
-        IObjectRepository& objectRepository, const std::string& objectName, bool loadImages,
+        IObjectRepository& objectRepository, const std::string& identifier, bool loadImages,
         const IFileDataRetriever* fileDataRetriever)
         : _objectRepository(objectRepository)
         , _fileDataRetriever(fileDataRetriever)
-        , _objectName(objectName)
+        , _identifier(identifier)
```

Comment:
Is this the new identifier, or the legacy one?

Reply:
I think it is new for JSON, old for DAT, but we only use it for logging at the moment.